features.html

```diff
@@ -1,322 +1,475 @@
+    <img src="/assets/features/navigation/search_everywhere.png" alt="search everywhere" width="750"/>
+    <p>In addition to <strong>Search Everywhere</strong>, you can separately call <strong>Go To</strong> class (<code>Ctrl+O / ⌘O</code>), symbol (<code>Ctrl+Alt+O / ⌥⌘O</code>), or file (<code>Ctrl+Shift+O / ⇧⌘O</code>).</p>
   </div>
-
-  <div class="feature half">
-    <h3>Quick documentation</h3>
-    <p><kbd>Ctrl+Q</kbd> shows docs for the symbol at caret.</p>
+  <div class="item_bottomtext2">
+    <p>To locate all the occurrences of any symbol in your code, place the caret on it and call <strong>Find Usages</strong>. The IDE understands symbol context, so you will not see textual matches in the results.</p>
+    <p>You can highlight the usages in the current file by pressing <code>Ctrl+Shift+F7 / ⇧⌘F7</code>, and then jump between them, or list all usages in a separate window using <code>Alt+F7 / ⌥F7</code>. You can filter and group the results, and jump back to the source code:
+    </p>
+    <img src="/assets/features/navigation/find_usages.png" width="550" align="center" alt="find usages"/>
   </div>
+  <div class="item_bottomtext3">
+    <p>To navigate through declarations/definitions and hierarchies in your code, use Go-To action for Declaration (<code>Ctrl+B / ⌘B</code>), Definition (<code>Ctrl+Alt+B / ⌥⌘B</code>), Type Declaration (<code>Ctrl+Shift+B / ⇧⌘B</code>), and Super Method (<code>Ctrl+U / ⌘U</code>), which navigates to a parent module.</p>
+    <p>There is also a handy <i>Go to Implementation</i> gutter icon to help you quickly switch between traits, types, and impls:
+    </p>
+    <img src="/assets/features/navigation/goto_impl_gutter.png" width="500" align="center" alt="go to implementation"/>
+  </div>
+  <div class="item_bottomtext4">
+    <p>All navigation actions are available inside macro calls when the <i>experimental macro expansion engine</i> is turned on (<strong>Settings / Preferences | Languages & Frameworks | Rust | Expand declarative macros</strong>):</p>
+    <img class="gif" data-wait="true" data-scope="gif-scope" style="max-width: 600px" style="max-height:370px"
+                          src="/assets/features/navigation/navigation_in_macros.png" data-gif="/assets/features/navigation/navigation_in_macros.gif"
+    alt="navigation inside macro calls"/>
+  </div>
+  <!--div class="item_bottomtext5">
+    <p>You can also explore the structure of the currently opened file in the <strong>Structure</strong> view (<code>Ctrl+F12 / ⌘F12</code>):</p>
+    <img src="/assets/features/navigation/file_structure.png" style="width: auto">
+  </div-->
 </div>
 
-<div class="feature-group">
-  <h2>Reading code</h2>
-  {% include feature-pic.html i="feature-reading" %}
-
-  <div class="feature half">
-    <h3>Quick documentation</h3>
-    <p><kbd>Ctrl+Q</kbd> shows docs for the symbol at caret.</p>
+<h2 id="analysis" align="left">Code analysis</h2>
+<div class="feature-section-3cols">
```

Comment:
Let's do not use three columns sections because in this case a column is too small